packages/hash/api/src/graphql/resolvers/ontology/model-mapping.ts

```diff
@@ -72,7 +72,10 @@ export const mapSubgraphToGql = (subgraph: Subgraph): SubgraphGql => {
       Object.entries(subgraph.vertices).map(([identifier, vertex]) => {
         switch (vertex.kind) {
           // These types are compatible with the Type System package's types
-          case "dataType" || "linkType" || "entity" || "link": {
```

Comment:
@benwerner01 I was stupid and the lint was trying to tell me and I ignored it 🙃.